specification/informatica/resource-manager/Informatica.DataManagement/stable/2024-05-08/openapi.json

```diff
@@ -0,0 +1,2249 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ServerlessRuntimes_CheckDependencies": {
+            "$ref": "./examples/ServerlessRuntimes_CheckDependencies_MaximumSet_Gen.json"
+          },
+          "ServerlessRuntimes_CheckDependencies_Min": {
+            "$ref": "./examples/ServerlessRuntimes_CheckDependencies_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Informatica.DataManagement/organizations/{organizationName}/serverlessRuntimes/{serverlessRuntimeName}/serverlessResourceById": {
+      "post": {
+        "operationId": "ServerlessRuntimes_ServerlessResourceById",
+        "tags": [
+          "ServerlessRuntimes"
+        ],
+        "description": "Returns a serverless runtime resource by ID",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "organizationName",
+            "in": "path",
+            "description": "Name of the Organizations resource",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_-]*$"
+          },
+          {
+            "name": "serverlessRuntimeName",
+            "in": "path",
+            "description": "Name of the Serverless Runtime resource",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_-]*$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Azure operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/InformaticaServerlessRuntimeResource"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ServerlessRuntimes_ServerlessResourceById": {
+            "$ref": "./examples/ServerlessRuntimes_ServerlessResourceById_MaximumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Informatica.DataManagement/organizations/{organizationName}/serverlessRuntimes/{serverlessRuntimeName}/startFailedServerlessRuntime": {
+      "post": {
+        "operationId": "ServerlessRuntimes_StartFailedRuntime",
```

Comment:
>ServerlessRuntimes_StartFailedRuntime [](http://example.com/codeflow?start=24&length=37)

Why this change from ServerlessRuntimes_StartFailedServerlessRuntime to ServerlessRuntimes_StartFailedRuntime